package/yast-update-alternatives.spec

```diff
@@ -0,0 +1,58 @@
+#
+# spec file for package yast-update-alternatives
+#
+# Copyright (c) 2016 SUSE LINUX Products GmbH, Nuernberg, Germany.
+#
+# All modifications and additions to the file contributed by third parties
+# remain the property of their copyright owners, unless otherwise agreed
+# upon. The license for this file, and modifications and additions to the
+# file, is the same license as for the pristine package itself (unless the
+# license for the pristine package is not an Open Source License, in which
+# case the license is the MIT License). An "Open Source License" is a
+# license that conforms to the Open Source Definition (Version 1.9)
+# published by the Open Source Initiative.
+
+# Please submit bugfixes or comments via http://bugs.opensuse.org/
+#
+
+Name:           yast2-update-alternatives
+Version:        0.1
```

Comment:
I suggest to follow yast versioning, which is now `3.1.X` so in your case `3.1.0` to be consistent.